функция.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace CourseApp
+{
+    using System;
+
+    public class Функция
+    {
+        public void Calculate(double a, double b, double x)
+        {
+            double top = Math.Sqrt(Math.Abs(a - (b * x)));
+            double bottom = Math.Sqrt(Math.Pow(Math.Log(10), 3));
+            double y = Math.Round(top / bottom, 3);

```

Comment:
Do round only on output

Reply:
double y = top / bottom;
 Console.WriteLine(Math.Round(y, 3));

Ok?

Reply:
Yes, but it will be good if you move output outside this method